### DISCUSSION TITLE
The Chrome browser extension is no longer working

### DISCUSSION THREAD

**Discussion by jhordanborges** [aclap-dev/video-downloadhelper#465] [License & Purchase requests]
### What's your issue?

According to the extension itself, the Chrome browser does not allow video transfers, and as such, the extension cannot include this feature in the Chrome version.

The extension itself recommends using Video DownloadHelper to download YouTube videos on the Firefox version. However, my license is for Chrome, and I would like to request a migration of my license from the Chrome extension.

### The 10 first digits (only 10) of your license number or email address

caa2069704

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
I sent you an email.
